packages/core/src/odata/common/uri-conversion/uri-value-converter.ts

```diff
@@ -40,7 +40,7 @@ export function convertToUriForEdmString(value: any): string {
 
 /**
  * Interface defining the methods of the URI converter.
- * The concrete implementations are created in odata/v2/uri-conversion/uri-value-converter.ts and odata/v4/uri-conversion/uri-value-converter.ts
+ * The concrete implementations are created in odata/v2/uri-conversion/uri-value-converter-v2.ts and odata/v4/uri-conversion/uri-value-converter-v2.ts
```

Comment:
When I refactored the names of the classes the IDE asked  `should I change also the filename?` I have no trong opinion here. In cases where we have a v2 parent folder I think we can get rid of this.

Reply:
@jjtang1985 what do you think?

Reply:
For me, we decided to use the class/interface name with `v2` suffix, because of the limit of the documentation. If the doc can live with the same file name, I would go without specifying the suffix, since the folder name has this information.

Reply:
I check with the documentation. If this works I will get rid fo the v2 v4 in folders.

Reply:
suffix removed if possible.